tests/library/inspector/cli-codegen-1.spec.ts

```diff
@@ -682,41 +682,20 @@ await page.Locator(\"#age\").SelectOptionAsync(new[] { \"2\" });`);
     expect(message.text()).toBe('2');
   });
 
-  const clickMultipleSelectOption = async (locator: Locator, withCtrlOrMeta = false) => {
-    const page = locator.page();
-
-    // Webkit can't click multiple select options
-    // https://github.com/microsoft/playwright/issues/32126
-    if (page.context().browser().browserType().name() === 'webkit') {
-      const elem = await locator.elementHandle();
-      const rect = await elem!.evaluate(e => {
-        return e.getBoundingClientRect()!;
-      });
-      if (withCtrlOrMeta)
-        await page.keyboard.down('ControlOrMeta');
-
-      await page.mouse.click(rect.x + rect.width / 2, rect.y + rect.height / 2);
-      if (withCtrlOrMeta)
-        await page.keyboard.up('ControlOrMeta');
-
-    } else {
-      await locator.click({ modifiers: withCtrlOrMeta ? ['ControlOrMeta'] : [] });
-    }
-  };
-
   test('should select with multiple attribute', async ({ openRecorder }) => {
     const { page, recorder } = await openRecorder();
 
     await recorder.setContentAndWait(`<select id="age" multiple onchange="console.log('[' + [...age.selectedOptions].map(x => x.value).join(',') + ']')"><option value="1">1</option><option value="2">2</option></select>`);
 
     const locator = await recorder.hoverOverElement('select');
     expect(locator).toBe(`locator('#age')`);
-    await clickMultipleSelectOption(page.getByRole('option', { name: '1' }));
+    await page.locator('select').focus();
+    await page.selectOption('select', ['1']);
```

Comment:
note, if you don't focus, it throws [here](https://github.com/microsoft/playwright/blob/c105de4436cc47b54a243bca2b44d12aa3a82d77/packages/playwright-core/src/server/injected/recorder/recorder.ts#L430) that `activeModel` is null.